geocoding/example/lib/plugin_example/geocode_page.dart

```diff
@@ -105,9 +107,11 @@ class _GeocodeWidgetState extends State<GeocodeWidget> {
                   onPressed: () {
                     locationFromAddress(_addressController.text)
                         .then((locations) {
-                      final output = locations[0].toString();
+                      var output = 'No results found.';
+                      if (locations != null && locations.isNotEmpty) {
```

Comment:
Same goes for the `locations` variable here (it will never be `null`):
```suggestion
                      if (locations.isNotEmpty) {
```